OrbitLinuxTracing/GpuTracepointEventProcessor.cpp

```diff
@@ -62,7 +62,7 @@ int GpuTracepointEventProcessor::ComputeDepthForEvent(
   auto it = timeline_to_latest_timestamp_per_depth_.find(timeline);
   std::vector<uint64_t>& vec = it->second;
 
-  for (int d = 0; d < vec.size(); ++d) {
+  for (std::size_t d = 0; d < vec.size(); ++d) {
```

Comment:
Is there any real benefit of using std::size_t instead of size_t? I just find the notation heavier.

Reply:
In theory it's more portable because you are guarantued that std::size_t is there but size_t might not. But on our target platforms that's not relevant. So it's just old habit. I can remove the `std::` if you prefer.

Reply:
Well, I do prefer a bare `size_t` because that's what I use, since you offered to make the change... :)

Reply:
I will check if the google coding guidelines say something about this.

Reply:
The guideline refers multiple times to size_t but never to std::size_t.

Reply:
They don't mention the std:: integer types at all but the non-scoped ones are all over the place. So I figure without `std::` is the way to go. I will change it! Thanks for pointing this out!

Reply:
> The guideline refers multiple times to size_t but never to std::size_t.

Ah sorry. Thanks. Now I read your answer. :-)